code/datums/movement/mob.dm

```diff
@@ -2,15 +2,6 @@
 	var/moving           = FALSE
 
 /mob/proc/SelfMove(var/direction)
-	///Mobs with slow turning take a move to turn in place.
-	//We will attempt to turn towards the target if our movement is off cooldown
-	if (slow_turning)
```

Comment:
This is code I added, its necessary for a lot of things, particularly lurkers and brutes.

Reply:
question not answered

Reply:
> question not answered

already returned.

Reply:
I don't see it here, it's still missing in the diff?

Reply:
> I don't see it here, it's still missing in the diff?

![image](https://user-images.githubusercontent.com/41364830/121162884-ff3f1900-c856-11eb-80ec-80aaecf09634.png)